packages/example/components/chains/btc/example.tsx

```
@@ -265,12 +283,13 @@ export default function BTCExample() {
               throw new Error('toAddress or amount is required');
             }
 
+            const network = await provider?.getNetwork();
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)